src/api/account.ts

```diff
@@ -224,12 +234,17 @@ export class Account {
    */
   async getAccountOwnedTokens(args: {
     accountAddress: AccountAddressInput;
+    minimumLedgerVersion?: string;
```

Comment:
eh, this is not a good type, it should be a AnyNumber, and converted to a string.